### PR TITLE
Emily's people.html

### PR DESCRIPTION
**Pull request by EmilyyyLiang**
First draft people.html is the latest one. I'll push a new commit after I have every photo. And please let me know if there are issues with the code that I haven't found so I can continue to change it. Also, I add Maheen's photo mistakenly, is it ok? If not I'll push a new commit.

### CODE REVIEW

File: people.html [hunk 1]
```diff
@@ -192,39 +192,40 @@ <h1>Rose Johnson</h1>
...
+
 		    <div class="item-5">
                         <a class="card">
-                            <div class="personPic" style="background-image: url('images/JustinLau.png')"></div>
+                            <div class="personPic" style="background-image: url('images/AndreasSoteriou.png')"></div>
                             <article>
-                                <h1>Justin Lau</h1>
-                                <span>Undergrad Researcher</span>
+                                <h1>Andreas Soteriou</h1>
+                                <span>Undergraduate Researcher</span>
```
Comment (ndattani) [inline people.html]: Why has Andreas changed from Masters to Undergrad?

File: people.html [hunk 2]
```diff
@@ -192,39 +192,40 @@ <h1>Rose Johnson</h1>
...
                     </div>
 		    <div class="item-6">
                         <a class="card">
-                            <div class="personPic" style="background-image: url('images/HenriAbensour.png')"></div>
+                            <div class="personPic" style="background-image: url('images/MichelleWatson.png')"></div>
                             <article>
-                                <h1>Henri Abensour</h1>
-                                <span>Undergrad Researcher</span>
+                                <h1>Michelle Watson</h1>
+                                <span>Undergraduate Researcher</span>
```
Comment (ndattani) [inline people.html]: Is this accurate, according to the instructions?

File: people.html [hunk 3]
```diff
@@ -192,39 +192,40 @@ <h1>Rose Johnson</h1>
...
                     </div>
 		    <div class="item-7">
                         <a class="card">
-                            <div class="personPic" style="background-image: url('images/AnantGupta.png')"></div>
+                            <div class="personPic" style="background-image: url('images/JustinLau.png')"></div>
                             <article>
-                                <h1>Anant Gupta</h1>
-                                <span>Undergrad Researcher</span>
+                                <h1>Justin Lau</h1>
+                                <span>Undergraduate Researcher</span>
```
Comment (ndattani) [inline people.html]: Sorry for writing "Undergraduate" in my instructions in the HPQC Discord server. We can stick with "Undergrad" which is what we had before. You can make that change, and then just commit that one change, and push it to the same fork of this repository, and the pull request should update automatically.

**Review by sahanas02 — APPROVED** [review]

**Review by sahanas02 — APPROVED** [review]

**Review by sahanas02 — APPROVED** [review]

**Comment by sahanas02** [comment]
.

**Comment by ndattani** [comment]
We don't generally allow "Force pushing", like you did, and we have chosen the GitHub setting in the main fork, to not allow force pushing ***at all***. However, it seems that GitHub still allows force-pushing in your own fork of the repository, which is what you did. It's still not ideal to use force-pushing even if GitHub hasn't disabled you from being able to do it. In this case, by force-pushing we have lost the entire history in this pull reuqet, such as what the dates and times were when certain changes were ***first*** made. This doesn't ***exactly*** apply to us: https://stackoverflow.com/q/7419244/1271772 but it conveys information relevant to what I've said in this paragraph.

Anyway, another issue is that the diffs in your commit make it appear that every single line of people.html was removed and replaced with something new, when in fact this was not really the case. It's therefore hard to see what the real diffs are:

image

File: people.html [hunk 4]
```diff
@@ -1210,3 +1343,1349 @@ <h1>Genevieve Laurier</h1><br>
...
+        <!--Initializes AOS script which is used to animate divs apppearing when scrolling down-->
+        <script src="Plugins/AOS/aos.js"></script>
+        <script>
+            AOS.init();
+        </script>
+
+</body>
+
+</html>
+>>>>>>> 9e030c5aadce3fe4f82d40f925a3a22219bddc11
```
Comment (ndattani) [inline people.html]: This is not right:
> `>>>>>>> 9e030c5aadce3fe4f82d40f925a3a22219bddc11`

File: people.html [hunk 5]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (ndattani) [inline people.html]: This is also wrong:
> `<<<<<< HEAD`

**Review by ndattani — CHANGES_REQUESTED** [review]

**Comment by ndattani** [comment]
I see that @maheenandcode replaced people.html with the version from the main branch, then added your desired new people.html back, so that I can now see the diffs between the original people.html and your proposed new one. 

However, when I clicked on the "Files changed" tab (https://github.com/HPQC-LABS/HPQC/pull/109/files), it says that you're removing 1212 lines and adding 1346 lines, which among other things, would give inaccurate statistics here: https://github.com/HPQC-LABS/HPQC/graphs/contributors

image

Do you know why this is happening?
Usually only the *actual* diffs would appear in the "Files changed" tab.
The problem seems to be this specific commit: https://github.com/HPQC-LABS/HPQC/pull/109/commits/62e0a11f044226f878d4871d82aa0152b8b10d36

Perhaps you can remove that specific commit?

**Comment by maheenandcode** [comment]
Will do that!

**Comment by ndattani** [comment]
The image of Karrar is now approved. We are just waiting for the fixed/cropped image of Mia.

**Pull request merged**